frontend/app/map/map.component.ts

```diff
@@ -1,9 +1,8 @@
 import { Component, ViewChild, AfterViewInit, NgZone } from "@angular/core";
 
-import { FacebookPageComponent } from "../facebook";
-import { EventComponent } from "../events";
 import { MapObject, MapObjectType, MapService } from "./";
 
+
```

Comment:
according to gulp lint:
"Consecutive blank lines are forbidden"
please remove one empty line